src/Bindings/ManualBindings.cpp

```diff
@@ -2343,6 +2344,34 @@ static int tolua_cClientHandle_SendPluginMessage(lua_State * L)
 
 
 
+static int tolua_cClientHandle_GetForgeMods(lua_State * L)
+{
+	cLuaState S(L);
+	if (
+		!S.CheckParamUserType(1, "cClientHandle") ||
```

Comment:
Use `CheckParamSelf` it checks if the class is a instance and is not nil. The nullptr check below can then be removed.